Formula/anka-scripts.rb

```diff
@@ -0,0 +1,27 @@
+class AnkaScripts < Formula
+  desc "Various scripts for setting up and testing Anka software on your local machine"
+  homepage "https://github.com/veertuinc/getting-started"
+  url "https://github.com/veertuinc/getting-started/archive/master.zip"
+  version "6ae6e30127cd5e22a63066f0e2bc9efcfca23936"
+  sha256 "1b476800e9c5704a2fd9a9db3cbc194f21a5f755d145b82ed36c19d713dd488f"
+
+  depends_on "jq"
+
+  def install
+    libexec.install Dir[".misc", "*"]
+    scripts = Pathname.glob "#{libexec}/**/*-*.bash"
+    scripts.each do |script|
+      wrap = bin + script.basename(script.extname)
```

Comment:
It looks as if the execution doesn't include the folders the scripts are in:

```
❯ cat /usr/local/bin/install-jenkins-on-docker 
#!/bin/bash
cd /usr/local/Cellar/anka-scripts/6ae6e30127cd5e22a63066f0e2bc9efcfca23936/libexec
./install-jenkins-on-docker.bash "$@"
```

This should be `./JENKINS/install-...`